03-specifying-sample-designs.Rmd

```diff
@@ -16,25 +18,25 @@ This chapter includes details on the first step - creating the survey object. Th
 
 ## Common sampling designs
 
-A sampling design is the method used to draw a sample. Both logistical and statistical elements are considered when developing a sampling design. When specifying a sampling design in R, the levels of sampling are specified along with the weights. Each record of a weight is constructed so that the particular record represents that many units in the population. For example, in a survey of 6th-grade students in the United States, the weight associated with each responding student reflects how many students that record represents. Generally, the sum of the weights sum to the population total though some studies have the sum of the weights sum to the number of respondent records.
+A sampling design is the method used to draw a sample. Both logistical and statistical elements are considered when developing a sampling design. When specifying a sampling design in R, the levels of sampling are specified along with the weights. Each record of a weight is constructed so that the particular record represents that many units in the population. For example, in a survey of 6th-grade students in the United States, the weight associated with each responding student reflects how many students that record represents. Generally, the sum of the weights sum to the population total though some studies have the sum of the weights sum to the number of respondent records. 
 
 Some common terminology across the designs are:
 
 - sample size, generally denoted as $n$, is the number of units selected
 - population size, generally denoted as $N$, is the number of units in the population
-- sampling frame is the list of units from which the sample is drawn
+- sampling frame, the list of units from which the sample is drawn
 
 ### Simple random sample without replacement
 
 - **Description**: The simple random sample (SRS) without replacement is a sampling design where a fixed sample size is selected from a sampling frame, and every possible subsample has an equal probability of selection.
 - **Requirements**: The sampling frame must include the entire population.
 - **Advantages**: SRS requires no information about the units apart from contact information.
 - **Disadvantages**: The sampling frame may not be available for the entire population. This design is not generally feasible for in-person data collection.
-- **Example**: Randomly students in a university from a roster provided by the registrar's office.
+- **Example**: Randomly students in a university from a roster provided by the registrar's office. 
 
 #### The math {-}
 
-The estimate for the population mean of variable $y$ is:
+The estimate for the population mean of variable $y$ is: 
 
 $$\bar{y}=\frac{1}{n}\sum_{i=1}^n y_i$$
 
```

Comment:
What do you think of adding a short explanation of what this means? Something like: "The symbol $\bar{y}$ represents the sample mean, which is obtained by summing up all the individual values of $y$ (represented by $y_i$) and dividing the sum by the total number of observations (denoted by $n$). This calculates the average value of the variable $y$ based on the given sample.